helm/url-shortener/custom-values.yaml

```diff
@@ -0,0 +1,129 @@
+kutt:
+  nameOverride: ""
+
+  fullnameOverride: ""
+
+  image:
+    repository: ghcr.io/kavetech/kutt
+    pullPolicy: Always
+    tag: "0.4"
+
+  imagePullSecrets:
+    - name: github-token-secret
+
+  serviceAccount:
+    create: false
+
+
+  service:
+    # -- Kubernetes service type
+    type: NodePort
+    # -- Kubernetes port where service is exposed
+    port: 3000
+
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.global-static-ip-name: url-shortener-ip-address
```

Comment:
This naming is not explained in the Readme, I would do some mention to it, in the link of Google-managed SSL Certificate.